# timestamp of synced GNSS measurement:
Header header

string child_frame_id

# a. position:
#geometry_msgs/Point position

# b.velocity:
geometry_msgs/Vector3 velocity